File: sistemaPCP/src/main/resources/webpcp/src/services/Solicitud_CertificacionService.js
import axios from 'axios';

export class Solicitud_CertificacionService{
    
    baseUrl="http://localhost:8080/api/solicitud_certificacion/";
    getAll(){
        return axios.get(this.baseUrl+"all").then(res => res.data)
      
    }
    
}

File: sistemaPCP/src/main/java/com/example/sistemaPCP/Service/api/Solicitud_CertificacionService.java
package com.example.sistemaPCP.Service.api;

import java.util.List;

import com.example.sistemaPCP.model.Solicitud_Certificacion;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface Solicitud_CertificacionService extends CrudRepository<Solicitud_Certificacion, Integer>{
    List<Solicitud_Certificacion> findAll();
    
}